sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/PropertiesAccessCorrectField.cs

```diff
@@ -706,4 +708,33 @@ public int ReadOnlyProperty
             get;
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/3441
+    public class Repro3441
+    {
+        private readonly List<int> _data = new List<int>();
+        public List<int> Data
+        {
+            get => _data;
+            // The setter in this case is only executed by a deserializer, but we don’t want to replace the field instance.
+            private set // Noncompliant
+            {
+                foreach (var item in value)
+                {
+                    _data.Add(item);
+                }
+            }
+        }
+    }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/3442
+    public class Repro3442 : System.Windows.Controls.Primitives.ButtonBase
```

Comment:
pedantic: not really a repro since it does the opposite, documents that the issue is not reproducible for dependency properties